apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/AccountController.java

```diff
@@ -302,17 +304,26 @@ public Response getAccountAssetsCount(
     // This GET returns two different responses (countAssetDTO or AccountAssetResponse)
     // it depends on the value of the asset parameter.
     public Response getAccountAssets(
-            @Parameter(description = "The account ID.", required = true, schema = @Schema(implementation = String.class)) @QueryParam("account") @NotNull AccountIdParameter accountIdParameter,
-            @Parameter(description = "The asset ID (optional).") @QueryParam("asset") @PositiveOrZero Long assetId,
-            @Parameter(description = "The height of the blockchain to determine the asset count (optional, default is last block).") @QueryParam("height") @PositiveOrZero Integer heightParam,
-            @Parameter(description = "Include asset information (optional).") @QueryParam("includeAssetInfo") @DefaultValue("false") boolean includeAssetInfo
+            @Parameter(description = "The account ID.", required = true, schema = @Schema(implementation = String.class))
+            @QueryParam("account") @NotNull AccountIdParameter accountIdParameter,
+            @Parameter(description = "The asset ID (optional).")
+            @QueryParam("asset") @PositiveOrZero Long assetId,
+            @Parameter(description = "The height of the blockchain to determine the asset count (optional, default is last block).")
+            @QueryParam("height") @PositiveOrZero Integer heightParam,
+            @Parameter(description = "Include asset information (optional).")
+            @QueryParam("includeAssetInfo") @DefaultValue("false") boolean includeAssetInfo,
+            @Parameter(description = "A zero-based index to the first asset ID to retrieve (optional)." )
+            @QueryParam("firstIndex") @DefaultValue("0") @PositiveOrZero int firstIndex,
```

Comment:
Need to validate max value of range [firstIndex;lastIndex] and trim it if specified range is greater than allowed in config

Reply:
added parser for first and last indexes;